zerver/migrations/0043_realm_filter_validators.py

```diff
@@ -14,13 +28,16 @@ class Migration(migrations.Migration):
         migrations.AlterField(
             model_name="realmfilter",
             name="pattern",
-            field=models.TextField(validators=[filter_pattern_validator]),
+            field=models.TextField(validators=[ignore_return_type(filter_pattern_validator)]),
```

Comment:
This can’t work in general, since Django compares the latest migration against the current model, but `ignore_return_type(filter_pattern_validator) is not ignore_return_type(filter_pattern_validator)`.

The only reason it appears to work here is that this isn’t the latest migration. But that means we could just as well remove the unused historical validator, and in fact this entire `AlterField` because the Django-side validator has no effect on the database.

Reply:
Did you mean that we could just remove this migration entirely? Updated the PR.

Reply:
I think we want to leave an empty migration with `operations = []`, so as not to anger `scripts/lib/check-database-compatibility.py`. (We could add more special cases to that check, but leaving the empty migration for now is simplest, and we can revisit this if Django provides better options for handling deleted migrations.)